raiden/api/python.py

```diff
@@ -1711,12 +1711,13 @@ def get_all_light_clients(self):
         light_clients = self.raiden.wal.storage.get_all_light_clients()
         return light_clients
 
-    def save_light_client(self,
-                          address,
-                          signed_password,
-                          server_name,
-                          signed_display_name,
-                          signed_seed_retry):
+    def store_matrix_light_client(self,
```

Comment:
it could, but i dont think the layer is related with storing data on the db. 

I think that we could move this to light_client_service that is in charge of those kind of operations

but i prefeer to do that on a future refactor

Reply:
moved to lc service